pontis-ui/src/components/Navigation/NavFooter.tsx

```diff
@@ -0,0 +1,150 @@
+import React from "react";
+import Link from "next/link";
+import { StyledExternalLink, StyledInternalLink } from "../common/StyledLink";
+
+interface FooterLink {
+  title: string;
+  href: string;
+  external: boolean;
+}
+
+interface FooterColumn {
+  header: string;
+  links: FooterLink[];
+}
+
+const content: FooterColumn[] = [
+  {
+    header: "Developers",
+    links: [
+      {
+        title: "Documentation",
+        href: "#",
+        external: false,
+      },
+      {
+        title: "View on Block Explorer",
+        href: "#",
+        external: false,
```

Comment:
Think these will both be external?

Reply:
Yes! I still need to provide the actual links, too.